examples/cdn-styles/README.md

```diff
@@ -11,7 +11,7 @@ The CDN hosted CSS file needs to be included in your HTML document. Replace the
 ```html
 <link
   rel="stylesheet"
-  href="https://unpkg.com/@shopify/polaris@4.0.0/styles.min.css"
+  href="https://unpkg.com/@shopify/polaris@4.0.0/styles.css"
```

Comment:
What part are you refering to?

https://unpkg.com/@shopify/polaris@4.0.0/styles.css currently exists, and will continue to do so.  This example should eventually be updated to use the new path (https://unpkg.com/@shopify/polaris@5.0.0/dist/styles.css) when v5 is released but we can't update these examples to use v5.0.0 until we actually release v5.0.0.

Reply:
Maybe I misunderstood when we talked, I thought you mentioned that linking unpkg css wasn't gonna work anymore

Reply:
It'll still work as long as you include the version in there and use the appropriate path for version 4 or version 5.

It will break if you don't include a version number:

https://unpkg.com/@shopify/polaris/styles.css works currently (it redirects to https://unpkg.com/@shopify/polaris@4.26.1/styles.css) but once we ship v5 it will break as it will redirect to https://unpkg.com/@shopify/polaris@5.0.0/styles.css which won't exist.

But imo that's a risk you take if you don't specify a major version number.